stl/inc/chrono

```diff
@@ -6202,6 +6163,128 @@ namespace chrono {
             }
         }
 
+        template <class _Ty>
+        void _Validate_specifiers(const _Chrono_spec<_CharT>& _Spec, const _Ty& _Val) {
+            // clang-format off
+            if constexpr (
+                _Is_specialization_v<_Ty, duration>
+                || is_same_v<_Ty, sys_info> || _Is_specialization_v<_Ty, time_point>
+                || _Is_specialization_v<_Ty, _Local_time_format_t>) {
+                return;
+            }
+            // clang-format on
+
+            if constexpr (_Is_specialization_v<_Ty, hh_mm_ss>) {
+                if (_Spec._Type == 'H' && _Val.hours() >= hours{24}) {
+                    _THROW(format_error("Cannot localize hh_mm_ss with an absolute value of 24 hours or more."));
+                }
+                return;
+            }

```

Comment:
Apart from `H`, my understanding is that `hh_mm_ss` is always valid (for it's specifiers).